src/enums.h

```diff
@@ -57,7 +57,11 @@ typedef enum {
     SWAPOUT_WRITING,
     /// StoreEntry is associated with a complete (i.e., fully swapped out) disk store entry.
     /// Guarantees the disk store entry existence.
-    SWAPOUT_DONE
+    SWAPOUT_DONE,
+    /// StoreEntry is associated with incomplete/failed disk store entry (marked for
```

Comment:
Please mimic the other descriptions in this enum to keep things consistent and reduce doubts. For example:

```suggestion
    /// StoreEntry is associated with a failed incomplete (i.e., not fully
    /// swapped out) disk store entry marked for eventual deletion.
    /// Guarantees the disk store entry existence.
```

Please check whether the existence guarantee is there and adjust the description as needed.